api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/PaymentTestDataFactory.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.dhsc.htbhf.claimant.testsupport;
+
+import uk.gov.dhsc.htbhf.claimant.entity.Payment;
+import uk.gov.dhsc.htbhf.claimant.entity.PaymentStatus;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import static uk.gov.dhsc.htbhf.claimant.testsupport.ClaimTestDataFactory.aValidClaim;
+
+public class PaymentTestDataFactory {
+
+    private static final UUID CLAIM_ID = aValidClaim().getId();
```

Comment:
Why have you done this rather than just build a new UUID? If you want to make sure it's the same as the generated Claim it won't because each call to `aValidClaim()` will generate a new UUID for each call to that method (as it generates a new object each time).

Reply:
Type has been changed to a Claim